pykoi/retrieval/vectordb/epsilla.py

```diff
@@ -0,0 +1,169 @@
+"""Vector store Epsilla module"""
+import os
+import numpy as np
+import types
+
+from typing import List
+from langchain.embeddings.base import Embeddings
+from langchain.schema import BaseRetriever, Document
+from langchain.embeddings import OpenAIEmbeddings
+from sklearn.decomposition import PCA
+from pyepsilla import vectordb
+
+from pykoi.retrieval.vectordb.abs_vectordb import AbsVectorDb
+
+class EpsillaRetriever(BaseRetriever):
+    def __init__(self, vector_db, embedding, limit):
```

Comment:
nit: add missing docstring and typehint for all missing methods to help improve readability of sphinx generated documents.

Reply:
please check